packages/cra-template-imodeljs/README.md

```diff
@@ -0,0 +1,20 @@
+# cra-template-typescript
```

Comment:
i would change this to imodeljs, as well as update the rest of the readme so we know its an imjs starter